minoc/src/main/java/org/opencadc/minoc/GetAction.java

```diff
@@ -99,12 +113,30 @@ public void doAction() throws Exception {
         
         Artifact artifact = getArtifact(artifactURI);
         HeadAction.setHeaders(artifact, syncOutput);
-        
+
```

Comment:
If a cutout is being performed, most of the header content changes. The above `HeadAction.setHeaders(...)` call is going to set wrong values in that case. So, that line has to move into the plain get output block (~line 124) and the cutout block should only set Content-Disposition and Content-Type (we'll worry about ops that change type when it happens).

There should be an intTest that verifies headers and it should assert that Content-Length, Content-MD5, and (probably) Content-Encoding are all not set for a cutout. And Digest, even though we don't currently set it (there's an issue for that).